src/mono/mono/metadata/unsafe-accessor.c

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#include "config.h"
+#include <glib.h>
+#include "mono/metadata/unsafe-accessor.h"
+#include "mono/utils/mono-compiler.h"
```

Comment:
Are these 2 files really needed ?

Reply:
It is more clear this way.

Reply:
@vargaz We probably don't need  `unsafe-accessor.c` right now, but we're going to need it when we start working on `UnsafeAsscessorKind.{Constructor,Method,StaticMethod}`  - we will need some helpers to pick the right overload if there are multiple methods with the same name, or multiple constructors. I think it's going to be complicated enough that putting all the code in a separate file will be worth it